tensorboard/plugins/interactive_inference/What_If_Tool_Notebook_Usage.ipynb

```diff
@@ -42,12 +42,12 @@
       },
       "cell_type": "code",
       "source": [
-        "#@title Install latest TensorFlow and the What-If Tool widget if running in colab {display-mode: \"form\"}\n",
+        "#@title Install the What-If Tool widget if running in colab {display-mode: \"form\"}\n",
```

Comment:
Would you mind making one more small changes to capitalize C in Colab before merging?

Reply:
done